cou/apps/app.py

```diff
@@ -135,6 +231,27 @@ def __str__(self) -> str:
             yaml.dump(summary, stream)
             return stream.getvalue()
 
+    @property
+    def expected_current_channel(self) -> str:
+        """Return the expected current channel based on the current OpenStack release.
+
+        Note that this is not necessarily equal to the "channel" property since it is
+        determined based on the workload version.
+        :return: The expected current channel for the application. E.g: ussuri/stable
+        :rtype: str
+        """
+        return f"{self.current_os_release}/stable"
+
+    def next_channel(self, target: OpenStackRelease) -> str:
```

Comment:
nit, optional: `next_channel` implies an N+1 channel. `target_channel` would be more generic

Reply:
It makes sense. Changed to `target_channel`